harvard-lts/drs-translation-service
# Embed Batch Builder

**Embeds batch builder into the image**
* * *

**JIRA Issue**: https://at-harvard.atlassian.net/browse/LTSK8S-756

# How should this be tested?

Set up the local container and run the tests as instructed in the [README](https://github.com/harvard-lts/drs-translation-service?tab=readme-ov-file#testing).  All tests should pass

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? no
- integration tests? no
